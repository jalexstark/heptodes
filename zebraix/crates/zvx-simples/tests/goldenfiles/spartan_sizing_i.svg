<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="500pt" height="500pt" viewBox="0 0 500 500">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 250 250 L 477.273438 22.726562 M 250 250 L 250 22.726562 M 250 250 L 136.363281 22.726562 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 250 250 L 477.273438 477.273438 M 250 250 L 250 477.273438 M 250 250 L 136.363281 477.273438 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 298.253906 204.546875 C 298.253906 208.277344 292.65625 208.277344 292.65625 204.546875 C 292.65625 200.8125 298.253906 200.8125 298.253906 204.546875 Z M 252.800781 204.546875 C 252.800781 208.277344 247.199219 208.277344 247.199219 204.546875 C 247.199219 200.8125 252.800781 200.8125 252.800781 204.546875 Z M 230.074219 204.546875 C 230.074219 208.277344 224.472656 208.277344 224.472656 204.546875 C 224.472656 200.8125 230.074219 200.8125 230.074219 204.546875 Z M 298.253906 295.453125 C 298.253906 299.1875 292.65625 299.1875 292.65625 295.453125 C 292.65625 291.722656 298.253906 291.722656 298.253906 295.453125 Z M 252.800781 295.453125 C 252.800781 299.1875 247.199219 299.1875 247.199219 295.453125 C 247.199219 291.722656 252.800781 291.722656 252.800781 295.453125 Z M 230.074219 295.453125 C 230.074219 299.1875 224.472656 299.1875 224.472656 295.453125 C 224.472656 291.722656 230.074219 291.722656 230.074219 295.453125 Z M 230.074219 295.453125 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 337.980469 156.160156 L 343.839844 162.019531 M 343.839844 156.160156 L 337.980469 162.019531 Z M 247.070312 156.160156 L 252.929688 162.019531 M 252.929688 156.160156 L 247.070312 162.019531 Z M 201.613281 156.160156 L 207.476562 162.019531 M 207.476562 156.160156 L 201.613281 162.019531 Z M 337.980469 337.980469 L 343.839844 343.839844 M 343.839844 337.980469 L 337.980469 343.839844 Z M 247.070312 337.980469 L 252.929688 343.839844 M 252.929688 337.980469 L 247.070312 343.839844 Z M 201.613281 337.980469 L 207.476562 343.839844 M 207.476562 337.980469 L 201.613281 343.839844 Z M 207.476562 337.980469 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 386.363281 109.492188 L 386.363281 117.78125 M 390.507812 113.636719 L 382.21875 113.636719 Z M 250 109.492188 L 250 117.78125 M 254.144531 113.636719 L 245.855469 113.636719 Z M 181.816406 109.492188 L 181.816406 117.78125 M 185.960938 113.636719 L 177.675781 113.636719 Z M 386.363281 382.21875 L 386.363281 390.507812 M 390.507812 386.363281 L 382.21875 386.363281 Z M 250 382.21875 L 250 390.507812 M 254.144531 386.363281 L 245.855469 386.363281 Z M 181.816406 382.21875 L 181.816406 390.507812 M 185.960938 386.363281 L 177.675781 386.363281 Z M 185.960938 386.363281 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 434.394531 68.183594 C 434.394531 71.617188 429.242188 71.617188 429.242188 68.183594 C 429.242188 64.746094 434.394531 64.746094 434.394531 68.183594 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 252.574219 68.183594 C 252.574219 71.617188 247.425781 71.617188 247.425781 68.183594 C 247.425781 64.746094 252.574219 64.746094 252.574219 68.183594 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 161.667969 68.183594 C 161.667969 71.617188 156.515625 71.617188 156.515625 68.183594 C 156.515625 64.746094 161.667969 64.746094 161.667969 68.183594 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 434.394531 431.816406 C 434.394531 435.253906 429.242188 435.253906 429.242188 431.816406 C 429.242188 428.382812 434.394531 428.382812 434.394531 431.816406 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 252.574219 431.816406 C 252.574219 435.253906 247.425781 435.253906 247.425781 431.816406 C 247.425781 428.382812 252.574219 428.382812 252.574219 431.816406 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 0%)" fill-opacity="1" d="M 161.667969 431.816406 C 161.667969 435.253906 156.515625 435.253906 156.515625 431.816406 C 156.515625 428.382812 161.667969 428.382812 161.667969 431.816406 "/>
</svg>
