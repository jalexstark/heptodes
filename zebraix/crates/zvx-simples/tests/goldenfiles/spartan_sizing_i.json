{
  "sizing_scheme": "SquareCenter",
  "base_width": 500.0,
  "base_height": 500.0,
  "axes_range": [
    5.0
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  0.0
                ],
                [
                  5.0,
                  5.0
                ]
              ],
              [
                [
                  0.0,
                  0.0
                ],
                [
                  0.0,
                  5.0
                ]
              ],
              [
                [
                  0.0,
                  0.0
                ],
                [
                  -2.5,
                  5.0
                ]
              ]
            ]
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  0.0
                ],
                [
                  5.0,
                  -5.0
                ]
              ],
              [
                [
                  0.0,
                  0.0
                ],
                [
                  0.0,
                  -5.0
                ]
              ],
              [
                [
                  0.0,
                  0.0
                ],
                [
                  -2.5,
                  -5.0
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Points": {
          "centers": [
            [
              1.0,
              1.0
            ],
            [
              0.0,
              1.0
            ],
            [
              -0.5,
              1.0
            ],
            [
              1.0,
              -1.0
            ],
            [
              0.0,
              -1.0
            ],
            [
              -0.5,
              -1.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "centers": [
            [
              2.0,
              2.0
            ],
            [
              0.0,
              2.0
            ],
            [
              -1.0,
              2.0
            ],
            [
              2.0,
              -2.0
            ],
            [
              0.0,
              -2.0
            ],
            [
              -1.0,
              -2.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Points": {
          "point_choice": "Plus",
          "centers": [
            [
              3.0,
              3.0
            ],
            [
              0.0,
              3.0
            ],
            [
              -1.5,
              3.0
            ],
            [
              3.0,
              -3.0
            ],
            [
              0.0,
              -3.0
            ],
            [
              -1.5,
              -3.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "centers": [
            [
              4.0,
              4.0
            ],
            [
              0.0,
              4.0
            ],
            [
              -2.0,
              4.0
            ],
            [
              4.0,
              -4.0
            ],
            [
              0.0,
              -4.0
            ],
            [
              -2.0,
              -4.0
            ]
          ]
        }
      }
    }
  ]
}
